src/main/java/mate/academy/springboot/swagger/service/ProductService.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.springboot.swagger.service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.swagger.model.Product;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+
+public interface ProductService {
+
+    Product create(Product product);
```

Comment:
```suggestion
    Product save(Product product);
```
better save() because this operation is used by you not only for the creation of the product but to update too